test/examples/Rust/.gitignore

```diff
@@ -0,0 +1 @@
+target/
```

Comment:
But I think we don't event need to add built packages to ignore because they will not be committed from CI

Reply:
Or you want to commit them?

Reply:
this is to avoid accidentally committing them from workstation.Say, `wasm-pack build --target nodejs` will produce `pkg` which will have a .gitignore file included, and so `pkg` is not tracked. I guess we want to explicitly ignore `target` dir as well

Reply:
Ok, maybe we can put all ignore patterns to root?

Reply:
We could if we are ok with the fact that we may need to add another pattern with new examples arriving into the repo. I would let those example 'projects' mind their own business. Even if at some point we decide to put the examples into a different folder (rather than: './test/examples') there will be no risk that the pattern in the root gitignore no longer covers the outputs produced locally